Tests/LeafKitTests/LeafTests.swift

```diff
@@ -423,8 +419,7 @@ final class LeafTests: XCTestCase {
     }
 
     // https://github.com/vapor/leaf/issues/105
-    // FIXME: Parser is not resolving expressions
-    func _testGH105() throws {
+    func testGH105() throws {
```

Comment:
`Tests/LeafKitTests/LeafTests.swift` - cleanup whitespace, re-enable an old GH issue test that now works